Program.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+
+namespace builder
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // Client Code
+            var whitePizzaBuilder = new WhitePizzaBuilder();
+            var director = new Director(whitePizzaBuilder);
+            director.makePizza();
+            Console.WriteLine(whitePizzaBuilder.GetPizza().DescribePizza());
+
+            //Bonus 1
+            var wheatPizzaBuilder = new WheatPizzaBuilder();
+            wheatPizzaBuilder
+            .BuildDough()
+            .BuildSauce()
+            .BuildTopping();
+            Console.WriteLine(wheatPizzaBuilder.GetPizza().DescribePizza());
+
+            //Bonus 2
+            var whitePizzaBuilder2 = new WhitePizzaBuilder();
+            director = new Director(whitePizzaBuilder2);
+            director.makePizza("White Ultra thin", "Tartar", new string[] { "Confectionary sugar", "Cinnamon", "Essence of kitten", "Bottled rainbow" });
```

Comment:
Good job entering the fields for the `Pizza`, but as written in the requirements you should ask the user to enter the input not to be hard coded.